src/EventManager.ts

```diff
@@ -73,9 +72,7 @@ export class EventManager {
 			events = [events];
 		} else if (!events) {
 			events = this.events.keys();
-		} else if (!isIterable(events)) {
```

Comment:
I'm still somewhat in favor of runtime checks like this even if the code is not part of a pipeline that comes from data storage (e.g., the API will be called in userland TypeScript or JavaScript code) since if the user is using JavaScript in their userland code they will now be faced with a less helpful error message.